src/components/SystemPreferences.vue

```diff
@@ -179,6 +188,26 @@ export default {
         @change="updatedCPU"
       />
     </div>
+    <hr>
+
+    <div class="checkbox">
+      <Checkbox
+        :value="telemetry"
+        label="Allow collection of anonymous statistics to help us improve Rancher Desktop"
+        @input="toggleTelemetry"
+      />
+      <v-popover placement="right">
+        <i class="icon icon-info" />
+        <span slot="popover">
+          Rancher Labs would like to collect a bit of anonymized information<br />
+          about the configuration of your installation to help make Rancher Desktop better.<br /><br />
```

Comment:
`<br><br>` should just be done with proper `<p>`s instead; it's actually semantic too (they're actually paragraphs).

Reply:
done -- I had copied this from dashboard, but, yeah, prefer the better markup